packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -16,6 +16,7 @@ import classnames from 'classnames';
 import { FC } from 'react';
 import { useSelector } from 'react-redux';
 import { Link, LinkProps } from 'react-router-dom';
+import { useExperiment } from 'calypso/lib/explat';
```

Comment:
You're starting to enter the Help Center inferno 🚒 

I'd be a bit cautious with Calypso imports inside `packages`. Especially ones that do network requests.

1. They often pull plenty of Calypso bytes into the editor. And our editor is already slow.
2. Because the Help Center runs on Atomic, and most of these requests don't work on Atomic. I know this one doesn't because it uses [wpcom.req](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/explat/internals/fetch-experiment-assignment.ts#L14) which blocks requests from Atomic.

cc @jessie-ross Hi 👋🏼  I saw your P2s on Explat and I wonder if you have any advice on this. TL;DR: Is there an easy way to run ExPlat on Atomic?

Reply:
Could I use [explat-client](https://github.com/Automattic/wp-calypso/blob/2c1818291dc4d7758f73f6f444df5bbcd6f0aebf/packages/explat-client/README.md#L11) directly in this case?

Reply:
Yes, that's the way, but _I think_ you'll still need to initiate an instance like [so](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/explat/index.ts#L11).

```js
const exPlatClient = createExPlatClient( {
	fetchExperimentAssignment,
	getAnonId,
	logError,
	isDevelopmentMode,
} );
``` 

And `fetchExperimentAssignment` depends on `wpcom.req`, which [requests](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/explat/internals/fetch-experiment-assignment.ts#L21) from  `'/experiments/0.1.0/assignments/calypso'` endpoint. 

This endpoint needs to be public (un-authed) and allows CORS, in which case you can just use standard `fetch`.

And guess what? I just checked and it is! I suppose you just need to pass the user ID as `anon_id` query param. @jessie-ross knows the answer.


so your `fetchExperimentAssignment` would be

```js
export default function fetchExperimentAssignment( {
	experimentName,
	userId,
}: {
	experimentName: string;
	userId: number | string | null;
} ): Promise< unknown > {
    const query = new URLSearchParams();
	query.add('experiment_name', experimentName);
    query.add('anon_id', userId);

	return fetch('https://public-api.wordpress.com/wpcom/v2/experiments/0.1.0/assignments/calypso' + query.toString())
}
```

Reply:
> Is there an easy way to run ExPlat on Atomic?

Ah, I see, I don't think we've run an experiment on atomic yet.

Will this code will always run on atomic, or will it also sometime run in simple?

If it the latter we'll need a dynamic approach that switches between techniques.

If it is the former, we could create an atomic client just as you mention.

> This endpoint needs to be public (un-authed) and allows CORS, in which case you can just use standard fetch.

> And guess what? I just checked and it is! I suppose you just need to pass the user ID as anon_id query param. @jessie-ross knows the answer.

This will likely work, but requests should ideally be authenticated - it provides extra checks, safety and would allow us to measure predicted churn, is there really no way to call the wp-api from atomic?

Reply:
There sure is! You just need to create a proxy endpoint in Jetpack or ETK. Something like [this](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-search.php#L64). 

This makes the request go through the site (same domain), `yoursite.wordpress.com/wp-json/blabla`, then to Jetpack (which signs it on the user behalf), then to public-api. Then, on the client, you can do [this](https://github.com/Automattic/wp-calypso/blob/trunk/packages/data-stores/src/support-queries/use-sibyl-query.ts#L18-L29).

Reply:
@yoavf Do you have the capacity to add the proxy endpoint as part of this work?

Reply:
@yoavf that works and I certainly don't want to block you, but I'd just double-check that this doesn't pull a ton of Calypso bytes into the Help Center. You can check by running `yarn build` in `apps/editing-toolkit`, then check the bundle size `apps/editing-toolkit/editing-toolkit-plugin/help-center/dist/help-center.min.js`. 

For comparision, you can pull a built ETK from `trunk` of any other PR, it's available under 

![image](https://github.com/Automattic/wp-calypso/assets/17054134/52bbaddf-eb65-4f5d-9180-3b03ead14470)

Then

![image](https://github.com/Automattic/wp-calypso/assets/17054134/5cd8d1bc-3a3c-407c-bdca-9c179629b206)

Currently, it's `976 388 bytes`